### ISSUE TITLE
Blogpost: "How to setup and run a Juno node?"

### ISSUE THREAD

**Issue by stranger80** [NethermindEth/juno#1393]
Publish a tutorial/walkthrough blogpost explaining the actions required to run a Juno node.
This is to be published alongside the P2P MVP release and should include P2P node operation aspects (eg. network setup/connectivity required?)

**Comment by stranger80** [comment]
Draft: https://www.notion.so/nethermind/Juno-p2p-overview-ef846befb43444e191db30120d18ea66

**Comment by JorikSchellekens** [comment]
I think this is stale, shall we close it?

**Comment by stranger80** [comment]
not really, this is still chased by Marketing team, and feels worth the time, as per https://www.notion.so/nethermind/Blog-post-part-1-How-to-juno-in-p2p-mode-b12caed7645f4bad80e0ffff9d112019 

The current idea is to publish the current state of P2P as "experimental feature" (not for prod use, and with caveats, etc.), and coordinate the publish with this blogpost - how does that sound?
